tests/test_mechanics.py

```diff
@@ -279,6 +279,31 @@ def test_discard_enchanted_cards():
 	assert not game.player1.hand
 
 
+def test_discover():
```

Comment:
Sure but I would strongly advocate holding off on developing that prototype Hijack API that was committed today, because we _are_ going to have to change the DSL and the way selectors work to optimize them and I think it is pointless coding an API now that will just get scrapped in a week.